test/sharness/t0027-rotate.sh

```diff
@@ -8,7 +8,10 @@ test_description="Test rotate command"
 test_rotate() {
         TEST_ALG=$1
 
-        test_init_ipfs
+        test_expect_success "ipfs init" '
+        export IPFS_PATH="$(pwd)/.ipfs" &&
+        ipfs init --profile=test -a=rsa -b=2048 > /dev/null
```

Comment:
Why RSA and not the defaults here? It doesn't seem like there should be any difference rotating from an RSA or Ed25519 key, but wanted to double check if you thought it was worth a test.

Reply:
Yep, defaults is fine: This is meant to initialize ipfs, not to test "rotating from one type to another".

Reply:
Fair enough, we can just add a test that rotates from `Ed25519 -> RSA -> Ed25519`

Reply:
Fixed.